CITATION.cff

```diff
@@ -0,0 +1,37 @@
+cff-version: 1.2.0
+authors:
+  - family-names: Vaidya
+    given-names: Gaurav
+    orcid: https://orcid.org/0000-0003-0587-0454
+  - family-names: Lapp
+    given-names: Hilmar
+    orcid: https://orcid.org/0000-0001-9107-0714
+title: "phyx.js"
+type: software
+license: MIT
+version: v1.0.1
+date-released: 2021-10-18
+identifiers:
+  - type: doi
+    value: 10.5281/zenodo.5576556
+    description: The concept DOI for JPhyloRef.
```

Comment:
I'm not sure I understand this in full and why it's here. For example, we have a DOI for the JOSS publication of JPhyloRef – why would that not be appropriate here? And conversely, remind me why we should have a DOI for JPhyloRef here at all?

Reply:
Ah I think I see it now. The description is simply wrong because this is the DOI for "the latest version" of the phyx.js code archived to Zenodo, and _not_ the "concept DOI for JPhyloRef". So this simply needs to be fixed, right?

Reply:
Correct! I fixed this description in f5ddb70.